### DISCUSSION TITLE
[Vote ended on 2022-10-12] Clarify inclusion requirements on temp files and editor configs

### DISCUSSION THREAD

**Discussion by briantist** [ansible-community/community-topics#146] [Votes done]
Issue:
- https://github.com/ansible-community/community-topics/issues/138

PR with new wording:
- https://github.com/ansible-collections/overview/pull/215

Vote `+1` to merge https://github.com/ansible-collections/overview/pull/215 or `-1` not to merge

**Comment by Andersson007** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by lowlydba** [comment]
+1

**Comment by ssbarnea** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by ssbarnea** [comment]
DevTools team will be very happy to quickly implement a test in ansible-lint for that, we only need a relatively stable list of banned files, so we do not endup having to patch it every other day. Openstack has one for editor config as they already have a policy to not allow such files to be tracked by git.

**Reply by felixfontein** [reply]
Please do not start a discussion here, but in #138. This discussion is **voting only**.

**Reply by briantist** [reply]
Replied here: https://github.com/ansible-community/community-topics/issues/138#issuecomment-1270028941

**Comment by russoz** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by mariolenz** [comment]
+1
